rdkit/Chem/Draw/UnitTestDraw.py

```diff
@@ -50,6 +47,51 @@ def setUp(self):
       IPythonConsole.UninstallIPythonRenderer()
     self.mol = Chem.MolFromSmiles('c1c(C[15NH3+])ccnc1[C@](Cl)(Br)[C@](Cl)(Br)F')
 
+    # For testMolsMatrixToLinear and testMolsMatrixToLinear (which test MolsMatrixToGridImage and its helper _MolsNestedToLinear)
+    s = "NC(C)C(=O)"
+    mol = Chem.MolFromSmiles(s)
+    natoms = mol.GetNumAtoms()
+    nbonds = mol.GetNumBonds()
+
+    # Set up matrix with oligomer count for the molecules
+    # Should produce this grid:
+    # NC(C)C(=O)
+    #                                NC(C)C(=O)NC(C)C(=O)
+    # NC(C)C(=O)NC(C)C(=O)NC(C)C(=O)                      NC(C)C(=O)NC(C)C(=O)NC(C)C(=O)NC(C)C(=O)
+    repeats = [[1], [0, 2], [3, 0, 4]]
+    
+    # Create molecule if there are 1 or more oligomers;
+    # otherwise, use None for molecule because drawing is distorted if use Chem.MolFromSmiles("")
+    self.mols_matrix = [[Chem.MolFromSmiles(s * count) if count else None for count in row] for row in repeats]
+            
+    self.legends_matrix = [[str(count) + " unit(s)" for count in row] for row in repeats]
+
+    def ith_item_list(nunits, items_per_unit, i=0):
+        return [((n * items_per_unit) + i) for n in range(nunits)]
+
+    self.highlightAtomLists_matrix = [[ith_item_list(count, natoms, 0) for count in row] for row in repeats]
+
+    # Another bond is created when molecule is oligomerized, so to keep the bond type consistent,
+    #   make items per unit one more than the number of bonds
+    self.highlightBondLists_matrix = [[ith_item_list(count, nbonds + 1, 1) for count in row] for row in repeats]
+
+    # Parametrize tests: In addition to supplying mols_matrix, supply 0-3 other matrices
+    # col labels: legends_matrix,       highlightAtomLists_matrix,      highlightBondLists_matrix      
+    #             Zero other matrices: 1 parameter set
+    self.param_sets = [(None,                None,                           None                          ), 
```

Comment:
I see what you're trying to do here, but don't think it actually improves readability, please format the code "normally" (i.e. just run `yapf` over it)